editor/layouts/layout-tools.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: 'Tools'
+sidebarTitle: 'Tools'
+description: 'There are several Layout tools available in Rive to build your responsive UI or content'
+---
+
+import { YouTube } from '/snippets/youtube.mdx'
+
+## Getting Started with Layouts
+
+There are a number of ways to start adding layouts to your designs.
+
+- Layout Tools in the Arrangement Tools menu
+- Wrap in Layout
+- Add Child Layout
+- Dragging and Dropping Layouts
+
+---
+
+### Layout Tools in the Arrangement Tools Menu
+
+A selection of layout options are available in the Arrangement Tools menu. These tools allow you to draw a Layout in 2 ways:
+
+1. When dragged directly onto the Artboard, it will create an Absolute Layout positioned relative to the Top Left of the Artboard.
```

Comment:
These should be bullet points. Typically you only use numbers for progression. Step 1, step 2, etc.